edgedb/_testbase.py

```diff
@@ -91,19 +96,21 @@ def _start_cluster(*, cleanup_atexit=True):
 
         help_args = [edgedb_server, "--help"]
         if sys.platform == 'win32':
-            help_args = ['wsl', '-u', 'edgedb'] + args
+            help_args = ['wsl', '-u', 'edgedb'] + help_args
 
-        if "--generate-self-signed-cert" in subprocess.getoutput(help_args):
+        if "--generate-self-signed-cert" in subprocess.run(
```

Comment:
So, on WSL you actually need to translate the cert file path back to Windows path, essentially the reverse of the status file translation above